kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/QuestionsSettings.vue

```diff
@@ -131,10 +132,13 @@
         props.setContinueAction(null);
       });
 
+      const questionCountIsEditable = computed(() => !props.settings.isChoosingManually);
```

Comment:
This is non-blocking, but I'm having a little difficulty wrapping my head around what the question count being editable means here. Why would not choosing manually impact the user's ability to adjust the number of questions? It probably makes sense, and I know that there are only so many variable name possibilities. I'm just a but stuck trying to reason through the relationship here

Reply:
Ah yes, this is litterally that, when we are choosing manually the user cannot modify the setting `questionCount`, as this is just set to the max number of questions we can select in that session, and they can just select any number of questions _up to_ that `questionCount`. In random mode, we need this to be editable to know how many questions does the user want to be randomly selected from the resource pool, but this doesnt happen in manual mode because the user express this just by selecting the questions they want in and thats it, no need to edit this `questionCount` number.

![image](https://github.com/user-attachments/assets/e13f5386-7b22-4316-854a-bd82d82dfa99)

Reply:
Ah, I see - thank you for clarifying!